tests/system/feature_store/test_feature_store.py

```diff
@@ -894,6 +894,53 @@ def test_ingest_partitioned_by_key_and_time(
         orig_columns.remove("patient_id")
         assert result_columns.sort() == orig_columns.sort()
 
+    def test_passthrough_feature_set(self):
+        name = f"measurements_set_{uuid.uuid4()}"
+        key = "patient_id"
+        measurements_set = fstore.FeatureSet(
+            name, entities=[Entity(key)], timestamp_key="timestamp", passthrough=True
+        )
+        source = CSVSource(
+            "mycsv",
+            path=os.path.relpath(str(self.assets_path / "testdata.csv")),
+            time_field="timestamp",
+        )
+
+        preview_pd = fstore.preview(
+            measurements_set,
+            source=source,
+        )
+        # preview does not do set_index on the entity
+        preview_pd.set_index("patient_id", inplace=True)
+
+        fstore.ingest(measurements_set, source)
+
+        # assert that online target exist (nosql) and offline target does not (parquet)
+        assert len(measurements_set.status.targets) == 1
```

Comment:
Probably we want to make sure the target is the correct target.

Reply:
it was checked two lines below :-)
changed order of lines to make it clearer.